Library/Formula/ola.rb

```diff
@@ -20,14 +21,15 @@ class Ola < Formula
 
   option :universal
   option "with-ftdi", "Install FTDI USB plugin for OLA."
+  option "with-rdm-tests", "Install RDM Tests for OLA."
 
   depends_on "pkg-config" => :build
   depends_on "cppunit"
```

Comment:
Okay thanks. Any plans to do so? If I understand your bottle concept correctly, will that not still drag the test deps in, despite the fact your unlikely to bother testing the binary?